dnsimple/services_test.go

```diff
@@ -0,0 +1,132 @@
+package dnsimple
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"reflect"
+	"testing"
+)
+
+func TestServices_servicePath(t *testing.T) {
+	if want, got := "/services", servicePath(nil); want != got {
+		t.Errorf("servicePath() = %v, want %v", got, want)
+	}
+
+	if want, got := "/services/1", servicePath(1); want != got {
+		t.Errorf("servicePath(1) = %v, want %v", got, want)
+	}
+
+	if want, got := "/services/1", servicePath("1"); want != got {
+		t.Errorf("servicePath('1') = %v, want %v", got, want)
```

Comment:
Please do not use single apex. Strings in Go are double apex. Single apex has a different meaning.
See https://blog.golang.org/strings